package.json

```diff
@@ -8,13 +8,15 @@
     "preview": "vite preview",
     "lint": "run-p lint:*",
     "lint:js": "eslint .",
-    "lint:markdown": "markdownlint-cli2 \"./**/*.md\" \"#node_modules\" \"#dist\"",
+    "lint:markdown": "markdownlint-cli2 \"./**/*.md\" \"#node_modules\" \"#dist\" \"#Bliss-Blissary-BCI-ID-Map\"",
```

Comment:
That was needed when the Blissary Map was a sub-module.  Since we now use `fetch()` to get the map, the sub-module is gone.  So have removed the lint exclusion rule.